File: python/ejer54.py
#mostrar usuarios mayor de edad
usuarios = [
    {"nombre": "Juan", "edad": 25}, 
    {"nombre": "María", "edad": 17},
    {"nombre": "Pedro", "edad": 30},
    {"nombre": "Ana", "edad": 15},
    {"nombre": "Luis", "edad": 20}
]
for usuario in usuarios:
    if usuario["edad"] >= 18:
        print(usuario["nombre"])
                

File: js/ejer54.js
//mostrar usuarios mayor de edad
const usuarios = [
    { nombre: 'Ana', edad: 22 },
    { nombre: 'Luis', edad: 17 },
    { nombre: 'María', edad: 19 },
    { nombre: 'Pedro', edad: 16 }
];

for (let i = 0; i < usuarios.length; i++) {
    if (usuarios[i].edad >= 18) {
        console.log(usuarios[i].nombre);
    }
}

